### DISCUSSION TITLE
Thumbnails

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#363] [General]
Hi,

Saw that not only antidos had an update depending, but also 'thumbnails'. I got an error with updating. No idea why. See my screenshot below.

![image](https://github.com/rejetto/hfs/assets/140949730/fcf1e1b2-85b3-4b86-8ff5-366a06040541)

**Comment by rejetto** [comment]
yes, "plugin rejetto/sharp missing", you have to install it

**Comment by rejetto** [comment]
but wait, don't install it if you are on Windows, as there are apparently problems with new version: https://github.com/rejetto/sharp/issues/1

**Comment by rejetto** [comment]
to avoid confusion i retired the update, until i have time to fix it

**Reply by PH5HP** [reply]
Hi,

Ok ... I was just planning to write another message with a screenshot full of errors... Then I saw this ... Ok ... I will wait before I try this out ...

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
i hope to have fixed it now

**Reply by PH5HP** [reply]
Hi,

No errors. Not sure what it does for the plugin 'thumbnails'.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
i wanted to split "thumbnails" plugin because updating always redownloads all the files, and 99% of it were files that don't change.
These files are for the most part  a library [named sharp](https://sharp.pixelplumbing.com/).
This way i'll be free to make slim and fast updates for the actual plugin.
Maybe i'll make a better solution in the future.

**Reply by PH5HP** [reply]
Hi,

Thank you for your explanation. Al clear now. Still trying to understand the plugin creation. Keep up the good work.

Kind regards,
Henk - ftp.ph5hp.nl
